manifests/state-ib-kubernetes/0070-deployment.yaml

```
@@ -55,6 +55,21 @@ spec:
                   - key: "node-role.kubernetes.io/control-plane"
                     operator: In
                     values: [ "" ]
+            {{- if .DeploymentNodeAffinity }}
```

nit: Consider using the following, to reduce nested conditions
```suggestion
{{- with .DeploymentNodeAffinity }}
  {{- with .PreferredDuringSchedulingIgnoredDuringExecution }}
    {{- range . }}
    - weight: {{ .Weight }}
      preference:
        {{ .Preference | toYaml | nindent 16 }}
    {{- end }}
  {{- end }}
{{- end }}
```